spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,16 @@
+require "rails_helper"
+
+RSpec.describe UsersController, type: :controller do
+  let(:user_one) {FactoryBot.create(:user)}
+
+  describe "GET #show/:id" do
+    it "should render show user" do
+      get :show, params: { id: user_one.id }
+      expect(response).to be_successful
+    end
+  end
+
+  after(:all) do
+    DatabaseCleaner.clean_with(:truncation)
+  end
```

Comment:
Ủa, sao cần code này em ơi, có config trong file rb cho tất cả chứ ai viết riêng thế này
File `spec_helper` 
`require "simplecov"
SimpleCov.start
RSpec.configure do |config|

  config.expect_with :rspec do |expectations|
    expectations.include_chain_clauses_in_custom_matcher_descriptions = true
  end

  config.mock_with :rspec do |mocks|
    mocks.verify_partial_doubles = true
  end

  config.expect_with :rspec do |c|
    c.syntax = [:should, :expect]
  end

  config.shared_context_metadata_behavior = :apply_to_host_groups

  config.before(:suite) do
  DatabaseCleaner[:active_record].strategy = :transaction
  DatabaseCleaner.clean_with(:truncation)
  end

  config.before(:each) do
    DatabaseCleaner.start
  end

  config.after(:each) do
    DatabaseCleaner.clean
    if Rails.env.test?
      FileUtils.rm_rf(Dir["#{Rails.root}/spec/support/uploads"])
    end
  end
end
`